cms/djangoapps/contentstore/views/item.py

```diff
@@ -574,7 +574,9 @@ def _duplicate_item(parent_usage_key, duplicate_source_usage_key, user, display_
         duplicate_metadata = {}
         for field in source_item.fields.values():
             if field.scope == Scope.settings and field.is_set_on(source_item):
-                duplicate_metadata[field.name] = field.read_from(source_item)
+                # Do not duplicate 'discussion_id' TNL-1754
+                if field.name != 'discussion_id':
```

Comment:
I'm actually curious why anyone would want to duplicate a discussion block at all.

The problem is that there's no way to change a `discussion_id` through the studio UI. Also, now that we generate the discussion id from a discussion module's location, duplicating a discussion module that doesn't have a discussion_id explicitly set will result in a discussion module with a new discussion id